_posts/2022-05-06-Setting-up-replication.md

```diff
@@ -0,0 +1,114 @@
+    ```
+
+3) In the `<DPAgent_root>/bin/` folder, run the `dpagent_servicedaemon.sh` file through the code editor and comment out the `pingAgent` function:
+
+    ![Alt text](../article_images/data_provisioning_agent1.png)
+
+    Run the `dpagent_env.sh` file in the same folder and comment out the lines starting with “-e”:
+
+    ![Alt text](../article_images/data_provisioning_agent2.png)
+
+4) In the root folder `<DPAgent_root>`, open the `dpagentconfig.ini` file, change the line:
+    ```
+    hana.useSSL=false
+    ```
+## Configuring the Data Provisionig Agent
+1) From the `<DPAgent_root>/bin/` folder, start the `Agent` with the command:
+    ```
+    ./agentcli.sh --configAgent
+    ```
+2) Start the Agent from section `2. Start or Stop Agent` 
+    ![Alt text](../article_images/data_provisioning_agent3.png)
+
+    Check the operability with the `Ping Agent` command
+    ![Alt text](../article_images/data_provisioning_agent4.png)
+
+3) Set up communication between Agent and SAP HANA Cloud.
+   Section 7. SAP HANA Connection
+   ![Alt text](../article_images/data_provisioning_agent3.png)
+
+   Select 1. Connect to SAP HANA Cloud via JDBC
+   ![Alt text](../article_images/data_provisioning_agent5.png)
+
+    a) Enter Use JDBC Encrypted Connection - set to true;
+
+    b) Enter Host Name `********-****-****-****-**********.hana.prod-eu20.hanacloud.ondemand.com`;
+
+    c) Enter Port Number (default database host port 443);
+
+    d) Enter `Agent Admin HANA User` - enter the name of the database administrator (DBADMIN)
+    ![Alt text](../article_images/data_provisioning_agent6.png)
+
+    e) Enter database password;
+
+    f) Enter Use Proxy Server - select false if not using;
+
+    g) Enter `HANA User Name` for `Agent Messaging` - a user name capable of sending an `Agent Message`, created in the `Database Explorer` in the SQL panel, with the command:
+
+    ```
+    CREATE USER <username> PASSWORD <password> NO FORCE_FIRST_PASSWORD_CHANGE SET USERGROUP DEFAULT;
+    ```
+
+    e) Enter user password.
+
+4) Register Agent section `8.Agent Registration пункт 1`. Point 1 Register Agent
+    ![Alt text](../article_images/data_provisioning_agent7.png)
+
+    a) Enter a new Agent name in the `Enter Agent Name` field
+
+    b) Enter your hostname or external IP in the `Enter Agent Host Name` field
+
+## Creating Remote Source
+
+1) On the `SAP HANA Database Explorer` page, select a database, select Agents in the `Catalog` folder and check that the registered Agent is connected and receives data.
+
+2) In the `Catalog` folder, select z and create using the GUI or using the SQL panel, the code can be found on the page:
+
+    https://help.sap.com/viewer/7952ef28a6914997abc01745fef1b607/2.0_SPS05/en-US/6ed502701abd4d1ca94d463d7dc6e99f.html 
+
+    in the section for a certain database there is a configuration code. Paste code in SQL Panel to `SAP HANA Database Explorer`.
```

Comment:
There is a configuration code in the section for a certain database. Paste code in SQL Panel to `SAP HANA Database Explorer`.